cmd/ticdc.toml

```diff
@@ -15,6 +15,15 @@ log-file = "/tmp/ticdc/ticdc.log"
 # log level (debug|info|warn|error) default: "info"
 log-level = "info"
 
+# Max log file size in MB (upper limit to 4096MB).
+log-file-max-size = 300
+
+# Max log file keep days. No clean up by default.
+log-file-max-days = 0
+
+# Maximum number of old log files to retain. No clean up by default.
+log-file-max-backups = 0
```

Comment:
The log related configuration in TiDB/PD is different, should TiCDC keep the same config style?

/cc @leoppro 

```
[log]
# Log level: debug, info, warn, error, fatal.
level = "info"

[log.file]
# Log file name.
filename = ""

# Max log file size in MB (upper limit to 4096MB).
max-size = 300

# Max log file keep days. No clean up by default.
max-days = 0

# Maximum number of old log files to retain. No clean up by default.
max-backups = 0
```

Reply:
agree, rest lgtm

Reply:
I have modified it. PTAL

Reply:
For the most cases, tiup sets log level and path via flags, so the compatibility issue is negligible.